doc/articles/toc.yml

```diff
@@ -107,6 +107,8 @@
         href: xref:Uno.Interop.MSAL
       - name: How to authenticate with OpenID Connect
         href: xref:Uno.Tutorials.OpenIDConnect
+      - name: How to Apple Sign-In Button
```

Comment:
@morning4coffe-dev "How to **add an** Apple Sign-In Button" ?
OR
"How to implement a sign-in with Apple Button" ?